src/adaptors/utils.js

```diff
@@ -513,3 +513,83 @@ exports.getTotalSupply = async (tokenMintAddress) => {
 
   return supplyInTokens;
 };
+
```

Comment:
I don't know if this is the right place for this?

I was thinking about moving it into Merkl adapter.